src/utils/secrets.ts

```diff
@@ -21,5 +24,14 @@ export function initSecrets(secrets: ApplicationSecrets) {
   if (secretsSingleton) {
     throw new Error('Secrets already initialised.');
   }
+  if (!secrets.APP && process.env.GAE_APPLICATION === 'github-health') {
```

Comment:
Would be nice if production flag was actually something like 'process.env.NODE_ENV === 'production' | 'staging''